.github/workflows/kind_e2e.yaml

```diff
@@ -130,25 +132,35 @@ jobs:
       run: |
         docker image prune -a -f
     - name: Determine latest Dapr Runtime version including Pre-releases
-      if: github.base_ref == 'master'
```

Comment:
By removing this, this will always be run, for running against release branches, it will always use the latest release version and not the pinned version.

Reply:
@mukundansundar The behavior is unchanged from the previous. This check is not removed but it has been moved down by few lines. 

The change here is-  
- it will  always calculate the "latest version" (to test dapr init without `runtime-version`) for all runs(schedule/prs)
- Run all other tests with "Pinned version" for all schedule runs.
- Run all other tests with "latest version" for all prs.